astro/src/content/blog/identity-verification-before-registration.mdx

```diff
@@ -0,0 +1,296 @@
+
+Navigate to <Breadcrumb>Customizations -> Lambdas</Breadcrumb>. Find and edit the `Fideo Self Service Registration Validation` lambda. Change the `FAIL_THRESHOLD` value at the top of the edit screen to `0.99`. This threshold lets almost any identity successfully register.
+
+![A screenshot of the admin lambda screen.](/img/blogs/identity-verification/modify-lambda.png)
+
+Save the new lambda code by clicking the <InlineUIElement>Save</InlineUIElement> button at the top right.
+
+To test this change: 
+
+* Open up the browser you used previously to register and visit [localhost:8080](http://localhost:8080).
+* Click the "To get started, create a new account." link.
+* Register with the `test@example.com` user.
+
+They were denied last time but now can register because of the relaxed threshold.
+
+![A screenshot of the admin lambda screen.](/img/blogs/identity-verification/registration-succeeded-with-test-user.png)
+
+## Measuring Success and Iteration
+
+You should measure the success and failure of your registration rates. The success rate can be tracked by capturing the [User Registration Create webhook](/docs/extend/events-and-webhooks/events/user-registration-create). Tracking the failures is a bit trickier. You have two options, explored below. 
+
+Either way, keep an eye on the failure rate and tweak your business logic to balance between letting users register and preventing fake registrations. The exact ratio depends on how costly each of these are.
+
+If the user is flagged, you can offer alternate registration mechanisms. You could let them register using a chatbot or customer support ticket. Tracking these lets you know if the identity verification logic needs to be changed.
+
+### Tracking Failures With Fetches
+
+You can make a fetch call for every failed registration.
+
+Here's a sample implementation which captures the failed email address. This is called when the risk threshold is reached.
+
+<RemoteCode 
+   url="https://raw.githubusercontent.com/FusionAuth/fusionauth-example-kickstart/refs/heads/main/identity-verification/lambdas/fideo.js"
+   language="javascript" 
+   tags="registrationfailfetch" />
+
+This requires you to set up a remote API.
+
+### Tracking Failures With Fetches
+
+You can also create an event log entry. In this case, you make a `console.log` function call when the risk threshold is reached. This creates an Event Log entry.
+
+<RemoteCode 
+   url="https://raw.githubusercontent.com/FusionAuth/fusionauth-example-kickstart/refs/heads/main/identity-verification/lambdas/fideo.js"
+   language="javascript" 
+   tags="registrationfaillog" />
+
+You can be notified of this via the Event Log webhook](/docs/extend/events-and-webhooks/events/event-log-create). You can also poll [the event logs](/docs/apis/event-logs#search-event-logs) for the prefix.
```

Comment:
```suggestion
You can be notified of this via the Event Log webhook](/docs/extend/events-and-webhooks/events/event-log-create). You can also poll [the event logs](/docs/apis/event-logs#search-event-logs) for the `'fideo-registration-failed '` prefix.
```

Reply:
I will add that in. Was trying to not add it because I didn't want to overly tie the javascript to the blog post, but agree it makes the search process clearer.